flow-model-generator/modules/flow-model-generator-core/src/main/java/io/ballerina/flowmodelgenerator/core/CodeAnalyzer.java

```diff
@@ -626,8 +639,14 @@ public void visit(FunctionCallExpressionNode functionCallExpressionNode) {
                     .codedata()
                         .symbol(functionName);
 
-            functionSymbol.typeDescriptor().params().ifPresent(params -> nodeBuilder.properties().functionArguments(
-                    functionCallExpressionNode.arguments(), params, documentationMap, functionSymbol.external()));
+            Optional<List<ParameterSymbol>> funcParams = functionSymbol.typeDescriptor().params();
+            if (funcParams.isPresent()) {
+                List<ParameterSymbol> params = funcParams.get().stream()
+                        .filter(p -> p.paramKind() != ParameterKind.INCLUDED_RECORD)
+                        .toList();
+                nodeBuilder.properties().functionArguments(
+                        functionCallExpressionNode.arguments(), params, documentationMap, functionSymbol.external());
+            }
```

Comment:
Can't we use `handleActionCallsParams`?

Reply:
+1

Reply:
updated in [commit](https://github.com/ballerina-platform/ballerina-dev-tools/pull/465/commits/d81e4d04436716c0ba7d36547223ba5181fd659c).